luigi/scheduler.py

```diff
@@ -224,10 +226,18 @@ def __init__(self, task_id, status, deps, resources=None, priority=0, family='',
         self.status_message = status_message
         self.scheduler_disable_time = None
         self.runnable = False
+        self.batchable = False
+        self.batch_id = None
 
     def __repr__(self):
         return "Task(%r)" % vars(self)
 
+    def is_batchable(self):
```

Comment:
The long term plan is just to replace this with `self.batchable` right? Can you write a comment that this is only for backward compatability (and perhaps include a date)